.travis.yml

```diff
@@ -13,18 +13,29 @@ addons:
     - libcurl4-openssl-dev
     - clang
     - nodejs
+    - gdb
```

Comment:
Please, move changes in travis.yml in separate commit.

Reply:
Yes, will move them to separate commit, but at the moment, i'm trying to get backtrace on crash in CI, so it will stay for a moment.